packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/ModalForm.js

```diff
@@ -103,7 +103,7 @@ const ModalForm = ({ onMetaChange, onSizeChange }) => {
     );
   });
 
-  const canResize = !NON_RESIZABLE_FIELD_TYPES.includes(attributes[selectedField].type);
+  const fieldSizeInfo = fieldSizes[attributes[selectedField].type];
```

Comment:
I guess you could destructure { resizable } 🤷‍♂️  if you wanted, but I also see how scoping it to fieldSizeInfo makes sense.